.github/workflows/build.yml

```diff
@@ -465,6 +468,11 @@ jobs:
             ./configure --prefix=/usr
             make
             make install
+            cd ../talloc-2.4.1
+            CC=gcc ./configure --prefix=/usr/local
+            make
+            make install
+            ln -s /usr/local/lib/pkgconfig/talloc.pc /usr/lib/64/talloc.pc
```

Comment:
I think this should be `ln -s /usr/local/lib/pkgconfig/talloc.pc /usr/lib/64/pkgconfig/talloc.pc` no?